### PR TITLE
Test isolation

### PR DESCRIPTION
**Pull request by audax**
Run all tests in row and islolate UI tests from non-UI tests.

### CODE REVIEW

**Comment by audax** [comment]
retest this

**Comment by julianseeger** [comment]
Crash reports are not removed between tests (missing isolation).
When one tests "crashes", all following tests start with an "do you want to report this crash" (or similar) message and won't work until a user manually reacts.

File: qabelbox/build.gradle [hunk 1]
```diff
@@ -8,7 +8,7 @@ spoon {
     // for debug output
     debug = true
     noAnimations = true
-    instrumentationArgs = ["package=de.qabel.qabelbox.ui"]
+    //instrumentationArgs = ["package=de.qabel.qabelbox.ui"]
```
Comment (julianseeger) [inline qabelbox/build.gradle]: //DELETEME

File: qabelbox/src/main/java/de/qabel/qabelbox/services/LocalQabelService.java [hunk 2]
```diff
@@ -138,6 +138,16 @@ public Contacts getContacts() {
         return getContacts(getActiveIdentity());
     }
 
+	/**
```
Comment (julianseeger) [inline qabelbox/src/main/java/de/qabel/qabelbox/services/LocalQabelService.java]: sed 's/\t/    /g'

File: qabelbox/src/main/java/de/qabel/qabelbox/services/LocalQabelService.java [hunk 3]
```diff
@@ -138,6 +138,16 @@ public Contacts getContacts() {
...
+     * Reset the persistence
+     *
+     * This should only be used in testing.
```
Comment (julianseeger) [inline qabelbox/src/main/java/de/qabel/qabelbox/services/LocalQabelService.java]: maybe move it to the LocalQabelServiceTester

Comment (audax) [inline qabelbox/src/main/java/de/qabel/qabelbox/services/LocalQabelService.java]: I can't, the ui tests need the real LocalQabelService…

**Comment by julianseeger** [comment]
Looks good but can't verify if the test "still" "run" without jenkins acknowledging it. If anyone can, I ack a merge.

**Pull request merged**